examples/irsa/irsa.tf

```diff
@@ -1,6 +1,6 @@
 module "iam_assumable_role_admin" {
   source                        = "terraform-aws-modules/iam/aws//modules/iam-assumable-role-with-oidc"
-  version                       = "2.14.0"
+  version                       = "~> v2.0"
```

Comment:
Is there any reason for this change ? If not, can you please drop it. It was bumped to improve TF 0.13 support. See https://github.com/terraform-aws-modules/terraform-aws-eks/pull/976 for more info.

Reply:
Reason was that I used IRSA along the way and noticed it didn't work. Then I changed the version constraint to what I found in aws iam modules readme and then it worked. 
But I can remove from this PR.

Reply:
Yep please. Can you please add this change in another PR and explain how this change fixed your problem.